app/src/main/java/org/mozilla/focus/browser/binding/BlockedTrackersBinding.kt

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.browser.binding
+
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.flow.mapNotNull
+import mozilla.components.browser.state.selector.findTabOrCustomTabOrSelectedTab
+import mozilla.components.browser.state.state.BrowserState
+import mozilla.components.browser.state.store.BrowserStore
+import mozilla.components.lib.state.helpers.AbstractBinding
+import mozilla.components.support.ktx.kotlinx.coroutines.flow.ifChanged
+
+/**
+ * Binding responsible for updating the total number of tracked blockers since install.
+ */
+class BlockedTrackersBinding(
+    store: BrowserStore,
+    private val tabId: String,
+    private val updateCount: (Int) -> Unit
+) : AbstractBinding<BrowserState>(store) {
```

Comment:
A `Middleware` may be a better choice and independent from the UI. With the `TrackerBlockedAction` it's also easier to count them. :)

Reply:
I preferred to handle this locally in order to not interfere with release trains for AC and Focus. But If you think using middleware would improve this, I can give it a go tomorrow.

Reply:
I meant I can use the binding locally, I thought you suggested implementing a middleware in AC for this.

Reply:
In theory you could have a middleware on `BrowserStore` (implemented in `focus-android` for now), that counts every `TrackerBlockedAction`, persists the count, and also dispatches an action on `AppStore` (which you can use for reading the state in the UI).